### PR TITLE
#154 - Entire package unpublish

### PR DESCRIPTION
**Pull request by genryxy**
Part of #154 
Implemented unpublishing of entire package

### CODE REVIEW

File: src/test/java/com/artipie/npm/http/UnpublishForceSliceTest.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                new RequestLine(
+                    RqMethod.DELETE, "/@hello%2fsimple-npm-project/-rev/undefined"
+                ),
+                Headers.EMPTY,
+                Content.EMPTY
+            )
+        );
+        MatcherAssert.assertThat(
+            "Meta file was deleted",
+            this.storage.exists(new Key.From("@hello/simple-npm-project", "meta.json"))
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/npm/http/UnpublishForceSliceTest.java:75]: @genryxy would not it be better to check that entire project directory was removed, not just `meta.json`?

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy just one suggestion, please, check

**Comment by genryxy** [comment]
@olenagerasimova thanks, corrected

**Review by olenagerasimova — APPROVED** [review]
@genryxy thanks, approving

**Comment by 0crat** [comment]
Job `gh:artipie/npm-adapter#171` is not assigned, can't get performer

**Pull request merged**